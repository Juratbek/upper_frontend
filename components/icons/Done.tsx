import { IIconProps } from './types';

export const DoneIcon = ({ width = 16, height = 16 }: IIconProps) => (
  <svg width={width} height={height} viewBox='0 0 16 16' fill='none'>
    <path
      d='M13.3327 4L5.99935 11.3333L2.66602 8'
      stroke='#334155'
      strokeWidth='2'
      strokeLinecap='round'
      strokeLinejoin='round'
    />
  </svg>
);
