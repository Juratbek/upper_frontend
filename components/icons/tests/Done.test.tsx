import { render, screen } from '@testing-library/react';

import { DoneIcon } from '../Done';

describe('Done icon', () => {
  it('renders the icon', () => {
    render(<DoneIcon />);
    const icon = screen.getByRole('icon');
    expect(icon).toMatchSnapshot();
  });
});
