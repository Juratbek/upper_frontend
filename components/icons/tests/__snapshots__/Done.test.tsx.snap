// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Done icon > renders the icon 1`] = `
<svg
  fill="none"
  height="16"
  role="icon"
  viewBox="0 0 16 16"
  width="16"
>
  <path
    d="M13.3327 4L5.99935 11.3333L2.66602 8"
    stroke="#334155"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
  />
</svg>
`;
